src/checkers/inference/InferenceLauncher.java

```diff
@@ -190,6 +189,13 @@ public void infer() {
 
         addIfTrue("--hacks", InferenceOptions.hacks, argList);
 
+        Mode mode = Mode.valueOf(InferenceOptions.mode);
+        if (InferenceOptions.ignoreDefaultAnnotations
```

Comment:
What is this doing? What is the difference between the field `ignoreDefaultAnnotations` and passing the option?